src/main/java/net/minecraftforge/common/BiomeManager.java

```diff
@@ -35,15 +36,8 @@
 public class BiomeManager
 {
     private static TrackedList<BiomeEntry>[] biomes = setupBiomes();
-
-    public static List<Biome> oceanBiomes = new ArrayList<Biome>();
-
-    static
```

Comment:
Additionally, we don't want to change the default contents of `oceanBiomes` in case someone was using them .